tech.json

```diff
@@ -799,7 +799,7 @@
         {
           "name": "canHeroShot",
           "requires": [],
-          "note": "The ability to follow a shot to prevent it from despawning before hitting its target."
+          "note": "The ability to time and position a shot, then follow it to move the camera to prevent it from despawning before hitting its target."
```

Comment:
I think I like the first description more.  More details could be added as extra.
```
The ability to follow a shot to prevent it from despawning before hitting its target.
This may include aiming between enemies or waiting for the shot to travel some distance before following it.
```